### ISSUE TITLE
Rename client directory to frontend

### ISSUE THREAD

**Issue by kardan** [akvo/akvo-lumen#302]
To match the "backend"

**Comment by kardan** [comment]
Will open a PR once https://github.com/akvo/akvo-lumen/issues/327 is merged into develop.

**Comment by kardan** [comment]
After more considerations, let's not do this. At some point we might get into the situation where we don't only have one ”server" then we might have a frontend & a backend server. Client will always be client.
